Classes/Utility/MiscellaneousUtility.php

```diff
@@ -92,6 +92,22 @@ public static function getIconForTemplate(Form $form) {
 	}
 
 	/**
+	 * Returns a generated icon file into typo3temp/pics
+	 * @param $originalFile
+	 * @param int $width
+	 * @param int $height
+	 * @return string
+	 */
+	public static function createIcon($originalFile, $width, $height) {
+		/** @var \TYPO3\CMS\Core\Imaging\GraphicalFunctions $image */
+		$image = GeneralUtility::makeInstance('\TYPO3\CMS\Core\Imaging\GraphicalFunctions');
```

Comment:
$className "\TYPO3\CMS\Core\Imaging\GraphicalFunctions" must not start with a backslash